extensions/amp-story/1.0/amp-story.js

```diff
@@ -2237,13 +2240,27 @@ export class AmpStory extends AMP.BaseElement {
 
     const pagesByDistance = this.getPagesByDistance_();
 
-    this.mutateElement(() => {
+    const preloadAllPages = () => {
       pagesByDistance.forEach((pageIds, distance) => {
         pageIds.forEach((pageId) => {
           const page = this.getPageById(pageId);
           page.setDistance(distance);
         });
       });
+    };
+
+    this.mutateElement(() => {
+      if (prioritizeActivePage) {
```

Comment:
Plz switch the condition to have a more readable early return

Reply:
Forgot to add the `return` but is fixed now!

Reply:
I meant

```
if (!prioritizeActivePage) {
  return preloadAllPages();
}
```